Makefile

```diff
@@ -47,15 +48,25 @@ else
 endif
 
 flake8:
-	flake8
+	flake8 --ignore=E203,E731,E402,W503,W504
```

Comment:
@ntoll, if you add these to the already existing ones in `setup.cfg`:
* They will automatically be applied in both Windows and UNIX-like OSes.
* The new ignores will be right along with the already existing ones, and support minor commonts.